pol-core/bscript/compiler/codegen/DataEmitter.cpp

```diff
@@ -35,10 +37,13 @@ unsigned DataEmitter::store( const std::byte* data, unsigned len )
 
 unsigned DataEmitter::append( const std::byte* data, unsigned len )
 {
-  unsigned position = data_section.size();
+  size_t position = data_section.size();
+  if ( position > std::numeric_limits<unsigned>::max() ) {
```

Comment:
Can the data still be addressed if position+len is beyond unsigned::max()? Should it perhaps check for that instead?

Reply:
Sure, we can go with that